runner/master_log.py

```diff
@@ -95,7 +95,15 @@ def human_readable_full_log(self, test_id: str, message: str) -> List[str]:
                 sender = EVENT_SENDER[event.type]
                 messages.append(f"{sender} ({event.timestamp}): {event.data['message']}")
             elif event.type == EventType.WAIT:
-                messages.append(f"SYSTEM ({event.timestamp}): Test '{event.test_id}' WAITING for {event.data['tokens']} tokens until {event.data['time']}")
+                log_str = f"SYSTEM ({event.timestamp}): Test '{event.test_id}' WAITING for "
+                if event.data['tokens'] > 0:
+                    log_str += f"{event.data['tokens']} TOTAL TOKENS, "
+                if event.data['time'] > datetime.now():
+                    log_str += f"for DATE {event.data['time']}, "
+                if event.data['percentage_finished'] > 0.0:
+                    log_str += f"{event.data['percentage_finished']}% of tests to be finished."
```

Comment:
- The `log_str` for `time` seems wrong ("for " should be removed?)
- It's assuming some specific ordering.
- It would be better to append the parts in a list and then do `", ".join(parts)`